docker-for-mac/index.md

```diff
@@ -175,28 +175,26 @@ For more information, see:
 #### Proxies
 
 Docker Desktop detects HTTP/HTTPS Proxy Settings from macOS and automatically
-propagates these to Docker and to your containers. For example, if you set your
+propagates these to Docker. For example, if you set your
 proxy settings to `http://proxy.example.com`, Docker uses this proxy when
 pulling containers.
 
-When you start a container, your proxy settings propagate into the containers.
-For example:
+Your proxy settings, however, will not be propagated into the containers you start.
+If you wish to set the proxy settings for your containers, you need to define
+environment variables for them, just like you would do on Linux, for example:
```

Comment:
Do we no longer set the cli configuration (https://github.com/docker/cli/pull/93)? Or did we not do so on Docker Desktop?

We should probably link to that section to explain how to set these proxies automatically for `docker run` and `docker build` (https://github.com/docker/cli/pull/2224) 

I think we have another location where we explain those (need to search)

Reply:
CC @mat007

Reply:
@thaJeztah https://github.com/docker/docker.github.io/pull/11071. Updated it to DockerID for clarity

Reply:
> Or did we not do so on Docker Desktop?

Maybe we did at some point (before my time), but it’s not the case anymore as I can’t find anything in the code base.

> We should probably link to that section to explain how to set these proxies automatically for docker run and docker build

Ah good point!